tutorials/FITS-tables/FITS_tables.ipynb

```diff
@@ -77,22 +91,28 @@
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
+   "metadata": {
+    "id": "me6HB0RrcpOA"
+   },
    "source": [
     "## Opening the FITS file and viewing table contents"
    ]
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
+   "metadata": {
+    "id": "Cp_kqCg4cpOA"
+   },
    "source": [
     "Since the file is big, let's open it with `memmap=True` to prevent RAM storage issues."
```

Comment:
```suggestion
    "It's possible to open a FITS file directly from the URL using the `Table.read` method (see [Getting Started with Table I/O](https://docs.astropy.org/en/stable/io/unified.html)). However, the Chandra event files contain multiple FITS table extensions. As part of this tutorial, we want you to get familiar with exploring FITS file contents, so we will open the FITS file directly with the `fits.open` function. \n \n
Since the file is big, let's open it with `memmap=True` to prevent RAM storage issues."
```